extensions/brew/src/components/actionPanels.tsx

```diff
@@ -12,6 +12,26 @@ import { CaskInfo } from "./caskInfo";
 import * as Actions from "./actions";
 import { useTerminalApp } from "./runInTerminal";
 
+const DebugSection = (props: { obj: Cask | Formula }) => (
+  <ActionPanel.Section>
+    <Action.Push
+      target={
+        <Detail
+          navigationTitle="Debug Info"
+          markdown={"```\n" + JSON.stringify(props.obj, null, 2) + "\n```"}
```

Comment:
```suggestion
          markdown={"```json\n" + JSON.stringify(props.obj, null, 2) + "\n```"}
```